cmd/route-emitter/main.go

```diff
@@ -73,8 +73,8 @@ func main() {
 	externalChan := make(chan struct{}, 1)
 	internalChan := make(chan struct{}, 1)
 	syncer := syncer.NewSyncer(clock, time.Duration(cfg.SyncInterval), logger)
-	externalScheduler := scheduler.NewRouteBroadcastScheduler(clock, natsClient, logger, "router", externalChan)
-	internalScheduler := scheduler.NewRouteBroadcastScheduler(clock, natsClient, logger, "service-discovery", internalChan)
+	externalScheduler := scheduler.NewRouteBroadcastScheduler(clock, natsClient, logger, "router", cfg.JitterFactor, externalChan)
```

Comment:
```suggestion
	externalScheduler := scheduler.NewRouteBroadcastScheduler(clock, natsClient, logger, "router", cfg, externalChan)
```

I'd prefer handing down the config and make it more flexible. If there are more properties to configure in the future, we won't have to change this again.